### DISCUSSION TITLE
Connecting to Wallet Stuck

### DISCUSSION THREAD

**Discussion by richkravh** [Chia-Network/chia-blockchain#6921] [Support]
I already did the following steps 

1. Go to .chia/mainnet/db and search for sqlite
2. Delete sqlite
3. Open CHIA GUI
But it's still stuck on CONNECTING TO WALLET, It would be good if CHIA DEV TEAM give some kind of progress text on CHIA GUI, so i can know what is exactly this gui doing while loading and showing only CONNECTING TO WALLET all over the time :<<

**Comment by loppefaaret** [comment]
that is likely the wrong database you removed, try this isntead:
WINDOWS WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

LINUX WALLET DATABASE RESET

Close down the Chia, and open up a terminal, and execute the below command:
```
mv ~/.chia/mainnet/wallet/db/block* ~/Desktop/
```
Then try opening up the Chia again
if it worked, you can remove the files we put on the Desktop!

**Comment by BrandtH22** [comment]
@richkravh , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
